code/modules/uplink/uplink_items/uplink_implants.dm

```diff
@@ -29,6 +29,22 @@
 	item = /obj/item/storage/box/syndie_kit/imp_freedom
 	cost = 5
 
+/datum/uplink_item/implants/hijack
+	name = "Hijack Implant"
+	desc = "An implant that will let you hack into the APCs on station, allowing you to control them at will and the machinery within those rooms."
+	item = /obj/item/implanter/hijack
```

Comment:
Storage box / Maybe throw this into a synda kit as well

Reply:
I'd rather it be like the stealth implant